dotcom-rendering/src/components/StickyLiveblogAsk.importable.tsx

```diff
@@ -0,0 +1,273 @@
+/**
+ * This is a hard coded support ask as a test to see if it works.
+ * If it does, it may become a more standard feature.
+ */
+
+import { css, ThemeProvider } from '@emotion/react';
+import { palette, space, textSans15 } from '@guardian/source/foundations';
+import {
+	LinkButton,
+	SvgArrowRightStraight,
+} from '@guardian/source/react-components';
+import type { Tracking } from '@guardian/support-dotcom-components/dist/shared/src/types/props/shared';
+import { useEffect, useMemo, useState } from 'react';
+import { submitComponentEvent } from '../client/ophan/ophan';
+import { shouldHideSupportMessaging } from '../lib/contributions';
+import { useAB } from '../lib/useAB';
+import { useAuthStatus } from '../lib/useAuthStatus';
+import { useCountryCode } from '../lib/useCountryCode';
+import { useIsInView } from '../lib/useIsInView';
+import { usePageViewId } from '../lib/usePageViewId';
+import { useConfig } from './ConfigContext';
+import type { ReactComponent } from './marketing/lib/ReactComponent';
+import {
+	addRegionIdAndTrackingParamsToSupportUrl,
+	createClickEventFromTracking,
+	createInsertEventFromTracking,
+	createViewEventFromTracking,
+} from './marketing/lib/tracking';
+
+const baseUrl = 'https://support.theguardian.com/contribute';
+
+// CSS Styling
+// -------------------------------------------
+
+// outer container
+const stickyLeft = css`
+	background: ${palette.neutral[100]};
+	position: sticky;
+	top: ${space[3]}px;
+`;
+const container = css`
+	width: 220px;
+	margin-left: ${space[5]}px;
+	margin-top: ${space[6]}px;
+
+	box-sizing: border-box;
+
+	/* Auto layout */
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: 0px;
+	gap: ${space[1]}px;
+`;
+const imageHeader = css`
+	background-color: ${palette.brand[400]};
+	text-align: center;
+	padding: 15px 0;
+	width: 220px;
+	height: 132px;
+`;
+// contains the text and cta
+const textBlock = css`
+	/* Auto layout */
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: ${space[2]}px;
+	gap: ${space[1]}px;
+`;
+const bodySection = css`
+	color: ${palette.neutral[0]};
+	${textSans15};
+	left: ${space[8]}px;
+`;
+const ctaSection = css`
+	color: ${palette.neutral[0]};
+	margin-top: ${space[3]}px;
+	margin-bottom: ${space[4]}px;
+`;
+const cta = css`
+	left: ${space[2]}px;
+	min-width: 100%;
+	width: 100%;
+	min-height: 30px;
+	height: 30px;
+`;
+const buttonStyles = {
+	textPrimary: palette.neutral[7],
+	backgroundPrimary: palette.brandAlt[400],
+	backgroundPrimaryHover: palette.brandAlt[300],
+};
+const contributionsTheme = {
+	button: buttonStyles,
+	link: buttonStyles,
+};
+
+function notForSaleSvg() {
```

Comment:
We usually tend to use `const` declarations rather than functions for JSX components and use capitalised names:

```ts
const NotForSaleSvg = () => (
  <svg>
    {/* … */}
  </svg>
)
```

Reply:
Thanks - fixed in skb/sticky-live-blog-support-asks